src/main/kotlin/org/opensearch/replication/action/changes/TransportGetChangesAction.kt

```diff
@@ -103,9 +103,7 @@ class TransportGetChangesAction @Inject constructor(threadPool: ThreadPool, clus
                 if(fetchFromTranslog) {
                     try {
                         ops = translogService.getHistoryOfOperations(indexShard, request.fromSeqNo, toSeqNo)
-                    } catch (e: ResourceNotFoundException) {
-                        fetchFromTranslog = false
-                    } catch (e: IllegalStateException) {
+                    } catch (e: Exception) {
                         fetchFromTranslog = false
                     }
```

Comment:
Can we log the exception at DEBUG or TRACE level?

Reply:
Done